pkg/api/v1/schema.go

```diff
@@ -97,6 +97,10 @@ type CPU struct {
 	Cores uint32 `json:"cores,omitempty"`
 	// Model specifies the CPU model inside the VMI.
 	// List of available models https://github.com/libvirt/libvirt/blob/master/src/cpu/cpu_map.xml.
+	// You also can specify special cases like "host-passthrough" to get the same CPU as the node
+	// and "host-model" to get CPU closest to the node one.
+	// You can find more information under https://libvirt.org/formatdomain.html#elementsCPU.
+	// Defaults to host-model.
```

Comment:
* s/Model specifies the CPU model/This specifies the CPU model
* s/You also can specify special cases/It is possible to specify special cases
* s/You can find more information under/For more information see

Reply:
> s/Model specifies the CPU model/This specifies the CPU model

Doc string in golang must start with the function or variable name